src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md

```diff
@@ -0,0 +1,205 @@
+- Previously, CockroachDB would hit an internal error when evaluating inserts into [`REGIONAL BY ROW`]({% link v24.1/table-localities.md %}#regional-by-row-tables) tables where the source is a [`VALUES`]({% link v24.1/insert.md %}#insert-default-values) clause with a single row and at least one boolean expression. The bug was introduced in v23.2.0 and is now fixed. [#125492][#125492]
+- Fixed a bug in [logging]({% link v24.1/logging.md %}) where an error code was misreported for canceled queries. This bug affected the [`SQL_PERF`]({% link v24.1/logging.md %}#sql_perf) (slow query logs) and [`SQL_EXEC`]({% link v24.1/logging.md %}#sql_exec) (sql exec logs) logging channels. [#124371][#124371]
+- Fixed a bug in which constant `LIKE` patterns containing certain sequences of backslashes did not become constrained scans. This bug has been present since v21.1.13 when support for building constrained scans from `LIKE` patterns containing backslashes was added. [#125507][#125507]
+
+<h3 id="v24-2-0-alpha-1-performance-improvements">Performance improvements</h3>
+
+- More efficient [query plans]({% link v24.1/cost-based-optimizer.md %}) are now generated for queries with [text similarity filters]({% link v24.1/trigram-indexes.md %}#how-do-trigram-indexes-work), e.g., `text_col % 'foobar'`. These plans are generated if the `optimizer_use_trigram_similarity_optimization` [session setting]({% link v24.1/session-variables.md %}) is enabled, which it is by default. This setting is disabled by default in previous versions. [#121973][#121973]
+- [Schema changes]({% link v24.1/online-schema-changes.md %}) that cause a data backfill, such as adding a non-nullable column or changing the primary key, will now split and scatter the temporary indexes used to perform the change. This reduces the chance of causing a write hotspot that can slow down foreground traffic. [#120736][#120736]
+- Multiple or large numbers of [grants]({% link v24.1/show-grants.md %}) on tables and types within one [transaction]({% link v24.1/transactions.md %}) now run faster. [#123414][#123414]
+- [Lock operations]({% link v24.1/architecture/transaction-layer.md %}#writes-and-reads-phase-1) are now removed from [query plans]({% link v24.1/cost-based-optimizer.md %}) when the optimizer can prove that no rows would be locked. [#124830][#124830]
+- Some privilege checks when scanning the [`crdb_internal.system_jobs`]({% link v24.1/crdb-internal.md %}) internal table now happen once before the scan, instead of once for each row. This will improve performance for queries that read from `crdb_internal.system_jobs`. [#117438][#117438]
+- Improved the initial range descriptor scan on startup. [Node startup]({% link v24.1/cockroach-start.md %}) should now be faster on larger stores [#123959][#123959]
+
+<h3 id="v24-2-0-alpha-1-contributors">Contributors</h3>
+
+This release includes 1110 merged PRs by 91 authors.
+We would like to thank the following contributors from the CockroachDB community:
```

Comment:
Lines 102-113 should be removed

From "we would like ..." to the end of the bulleted list of folks